simdjson-flags.cmake

```diff
@@ -106,15 +108,11 @@ endif()
 # Workaround for https://gitlab.kitware.com/cmake/cmake/issues/15415#note_633938:
 function(export_private_library NAME)
   install(TARGETS ${NAME}
-    EXPORT ${NAME}-config
+    EXPORT simdjson-config
```

Comment:
So the fix is to make simdjson-headers and simdjson-source install as part of simdjson? Fascinating. I feel like we should just drop export_private_library at this point. I only added it because I just couldn't figure out how to make the simdjson target install, and this made it work. They are supposed to be private anyway, I just don't know cmake well enough.

If we can get to the point where the only export from simdjson is simdjson (the library), that sounds just about perfect.

Reply:
Yeah, right now they are exported as part of the simdjson target. I will be looking for a way to get rid of those as well